pyat/at/tracking/patpass.py

```diff
@@ -93,6 +93,13 @@ def patpass(ring, r_in, nturns=1, refpts=None, losses=False, pool_size=None,
         coordinates at which the particle is lost. Set to zero for particles
         that survived
     """
+    start_method = kwargs.pop('start_method','spawn')
+    if start_method is not None:
+        #try:
+        multiprocessing.set_start_method(start_method)
+        #except RuntimeError:
+        #   pass
+
     if refpts is None:
```

Comment:
No ! `set_start_method` must be called ONCE, at the beginning. Here you may call it several times. It must be called by the user after importing at. I still do not see any good reason to deviate from python' default. Except python experts, nobody understands what start_method means, so leave it for experts.

Reply:
This was just for testing, all the tests passed it seems, there are really things I don't understand.

But I disagree with you it should at least be an input argument, default will be `None` i.e. python default. The help will explain what it is and user can easily change it if he faces runtime errors

Reply:
A good reason is speed, 'fork' is faster and this is well documented